lib/libvfio-user.c

```diff
@@ -830,7 +851,7 @@ MOCK_DEFINE(should_exec_command)(vfu_ctx_t *vfu_ctx, uint16_t cmd)
             return false;
         }
     } else if (device_is_stopped(vfu_ctx->migration) &&
-               cmd != VFIO_USER_DIRTY_PAGES) {
+               !(cmd == VFIO_USER_REGION_READ || cmd == VFIO_USER_REGION_WRITE || cmd == VFIO_USER_DIRTY_PAGES)) {
```

Comment:
please add a helper instead of this very long line

Reply:
ok